api/user/lookup.py

```
@@ -1,30 +1,35 @@
-        [
-            SUG.REQ.PST["Uid"]
-        ]
-    )[0])
-    return SUG.REQ.RES.get()
\ No newline at end of file
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Ruff (0.11.9)

`from website import *` used; unable to detect undefined names

(F403)